docs/remote/troubleshooting.md

```diff
@@ -516,8 +516,10 @@ The SSH extension provides a command for cleaning up the VS Code Server from the
 If you want to run these steps manually, or if the command isn't working for you, you can run a script like this:
 
 ```bash
-kill -9 `ps ax | grep "remoteExtensionHostAgent.js" | grep -v grep | awk '{print $1}'`
-kill -9 `ps ax | grep "watcherService" | grep -v grep | awk '{print $1}'`
+# Kill server processes and clean up related files
+kill -9 `ps ax | \grep vscode-server | \grep -v grep | awk '{print $1}'`
+rm $(\ls ~/.vscode-server/.$(\ls -1t ~/.vscode-server/bin | head -1).*)
```

Comment:
> Also add a separate command to clean up the most recent .$hash.{pid,log,token} files.

This seems like an unnecessary command. These files are generated on installation of the VS Code server and are overwritten to be correct every time the server is installed or started.

What was the scenario in which removing the most recent of these files were helpful?

Reply:
This is needed if you want to stop the server, but not delete the whole `~/.vscode-server` folder. For example, let's say you are on a slow connection and want to avoid having to download the server again.

If I only kill the process, I do not delete these files,  and I then try to reconnect, it seems the server looks in these files for its PID and is confused because the processes were killed; so nothing works.

I we remove the whole folder I agree this command is not necessary, but I think it's nice to be able to kill the process and restart the server without downloading it again. Then this separate command is needed.

Reply:
I just pushed an updated version with that justification in the commit message.

Reply:
Hm that seems like a bug if you've successfully killed the VS Code Server process then it will try to find and verify the running process at that PID, if that fails it will start the VS Code Server process again and overwrite the `<hash>.pid` file. If you're seeing something other than this behavior could not create an issue for the behavior you're seeing in https://github.com/microsoft/vscode-remote-release

Reply:
I've removed the additional command in the latest version, I'll mark this conversation as resolved :)